modules/tensor_mechanics/doc/content/source/materials/ADIsotropicPlasticityStressUpdate.md

```diff
@@ -1,44 +0,0 @@
-# Isotropic Plasticity Stress Update
```

Comment:
We might still need this file...

Reply:
Do we need this?  It passes the sqa test.

Reply:
ok, I guess not then